Browser_IDE/editorMain.js

```diff
@@ -769,23 +777,32 @@ async function runProgram(){
         let compilableFiles = await findAllCompilableFiles();
         let sourceFiles = await findAllSourceFiles();
         if (compilableFiles.length == 0) {
+        	currentNotification.deleteNotification(); 
             displayEditorNotification("Project has no source files! In a "+activeLanguage.name+" project, valid source files end with:</br><ul>"+
                 activeLanguage.compilableExtensions.map((s)=>"<li>."+s+"</li>").join("")+"</ul>",
```

Comment:
I know you didn't add this line of code, but we could improve readability by setting the message outside the function call.